IssueTracker/IssueDb/dbo/StoredProcedures/Issue/spIssue_NextPreviewStatus.sql

```diff
@@ -0,0 +1,29 @@
+﻿CREATE PROCEDURE [dbo].[spIssue_NextPreviewStatus]
+	@Id int,
+	@Method varchar(8),
+	@StatusId int
+AS
+begin
+	IF (@Method = 'next')
+	begin
+		if(@StatusId = 4)
+		begin
+			print 'This issue is max level of status!'
+		end
+		else
+		begin
+			Update Issue set StatusId = (@StatusId + 1) where Issue.Id = @Id
```

Comment:
what happens if the status ids are out of order?

Reply:
i change this procedure